test/testdrive/recorded-views.td

```diff
@@ -49,9 +49,16 @@ $ kafka-ingest format=avro topic=recorded-views schema=${recorded-views} publish
 
 > CREATE SINK sink1 FROM v1
   INTO KAFKA BROKER '${testdrive.kafka-addr}'
-  TOPIC 'recorded-view-sink'
```

Comment:
Why did this file need to change?

Reply:
I pulled in a commit from @philip-stoev that made this test less flaky because it was failing on my PR
https://github.com/MaterializeInc/materialize/pull/13605/commits/e13de6bb02fec4d112b5bd4b22358417491004e0